.travis.yml

```diff
@@ -31,12 +36,24 @@ before_script:
     # Get last remaining tags, if any.
     - git fetch --tags origin
 
-    - make extracheck
-
-script:
-    - make coverage
-
-    - make
+jobs:
```

Comment:
The new build matrix. 
For now there are two stages, which means that a new VM is booted for each step. Personally, I think we maybe should keep it all in one stage. It will be faster that way. Thoughts?

Reply:
IMO we should keep it in 2 stages to avoid having required dependencies present in the test environment that we would otherwise not know about - should be the exception, but still better to catch it in a test than on a users machine

Reply:
+1 to what @mzedel said!